library/src/main/java/com/getstream/sdk/chat/utils/MessageInputController.java

```diff
@@ -150,27 +151,29 @@ public void onClickCloseBackGroundView() {
         binding.getRoot().setBackgroundResource(0);
         messageInputType = null;
         commandMentionListItemAdapter = null;
+        configAttachmentButtonVisible(true);
     }
 
     // endregion
 
     // region Upload Attachment File
 
-    private void configSelectAttachView(List<Attachment> editAttachments, boolean isMedia) {
+    private void configSelectAttachView(@Nullable List<Attachment> editAttachments, boolean isMedia) {
         binding.setIsAttachFile(!isMedia);
         
         selectedAttachments = (editAttachments != null) ? editAttachments : new ArrayList<>();        
         localAttachments = getAttachmentsFromLocal(isMedia);
         
         ((Activity) context).runOnUiThread(() -> {
-            if (!localAttachments.isEmpty()){
-                setAttachmentAdapters(isMedia);
+            if (editAttachments == null || editAttachments.isEmpty()){
+                if (!localAttachments.isEmpty()){
```

Comment:
please always avoid if-not when possible it makes it much easier to read code. ie

```java
if (selectedAttachments.isEmpty()) {
    Utils.showMessage(context, context.getResources().getString(R.string.stream_no_media_error));
    onClickCloseBackGroundView();
} else {
    setAttachmentAdapters(isMedia);
}
```